src/main/java/com/salesforce/dynamodbv2/mt/util/CachingAmazonDynamoDbStreams.java

```diff
@@ -217,6 +221,17 @@ public Builder withMaxIteratorCacheSize(int maxIteratorCacheSize) {
             return this;
         }
 
+        /**
+         * Cache to use for describe stream calls.
+         *
+         * @param describeStreamCache The cache to use for describe stream
+         * @return this Builder.
+         */
+        public Builder withDescribeStreamCache(Cache describeStreamCache) {
+            this.describeStreamCache = describeStreamCache;
+            return this;
+        }
```

Comment:
Do you still need to expose this?

Reply:
https://github.com/salesforce/mt-dynamo/pull/509/files#diff-2a4b73d2ee26b7ce82d49248a4931568 `MtAmazonDynamoDbStreamsBaseTest ` uses it in the test.  Before there was a getter for the test, in the past we discussed adding a builder instead.  In both options (getter or builder method), the test need a way to get to the cache.